app/models/episode.rb

```diff
@@ -210,4 +210,14 @@ def media_files
   def audio_files
     media_files
   end
+
+  def set_adzerk_keyword
+    return unless published? && adzerk_keyword.nil?
+    identifiers = []
+    [:title, :published_at, :guid].each do |attr|
+      # Adzerk does not allow commas or colons in keywords
+      identifiers << self.send(attr).to_s.slice(0, 10).gsub(/[:,]/,'')
+    end
+    self.adzerk_keyword = identifiers.join('_')
```

Comment:
Skipping the dashes, putting date first, and giving extra space to title sounds good -- I'll check with Kali tomorrow to see how that combo sounds too.